test/image/image_types/test_insert_images.py

```diff
@@ -116,6 +116,14 @@ def test_insert_bmp(tmp_path):
     assert_pdf_equal(pdf, HERE / "image_types_insert_bmp.pdf", tmp_path)
 
 
+def test_insert_jpg_icc(tmp_path):
```

Comment:
Could you please add a unit test when the same image is added twice in the same doc?
I'd like to ensure that a single `PDFICCPObject` object gets inserted in that case 😊

Reply:
done, I also added the case where the same ICC profile is included in multiple images